### PR TITLE
chore: roadmap updates

### PR DESCRIPTION
**Pull request by avishnu**

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -11,44 +11,45 @@ OpenEBS follows a lean project management approach by splitting the development
...
 
 ## Mayastor Roadmap : 2024 Phase-2
 - Forward facing planned release date, Release version numbers and feature priorities are subject to change as the project Maintainers/Leadership/community continuously update and adjust the **Release Feature bundling Strategy** to react to K8s industry movements, trends and our community influence.
 
 |  ID  | Feature name                   | Description and user story                                            | Release, links, tracking issue, GitHub repo                                                   |
 | :--- | :----------------------------- | :--------------------------------------------------------------------- | :------------------------------------------------------------------------------------------ |
-| 1    | Multi-replica volume snapshot and CSI cloning | Able to take consistent snapshots across all available replicas of a volume                                    | Pri 1 /  Rel: (Q1 2024)    |
+| 1    | Multi-replica volume snapshot and CSI cloning | Able to take consistent snapshots across all available replicas of a volume                                    | Pri 1 /  Rel: (Q3 2024)    |
```
Comment (tiagolobocastro) [inline ROADMAP.md:23]: Do we need to mark this as done now?

File: ROADMAP.md [hunk 2]
```diff
@@ -11,44 +11,45 @@ OpenEBS follows a lean project management approach by splitting the development
...
+| 6.5  | NDM enabled                                   | NDM support for all Local-PV dependant services                                                                 | Pri 2 /  Rel: (Dropped, Archived)   |
+| 7    | HyperLocal-PV Data-Engine                     | Dynamically provision a non-replicated PV of Local-PV type via SPDK blobstor LVol as storage + NVMe target device |  Pri 2 /  Rel: (2025)   |
+| 7.1  | HyperLocal-PV : UBlock mode                   | Non-replicated PV of Local-PV type via UBlock kernel integration to SPDK blobstor LVol as storage                  |  Pri 2 /  Rel: (2025)   |
+| 7.2  | HyperLocal-PV : PCIe mode                     | Non-replicated PV of Local-PV type via PCIe-based NVMe kernel integration to SPDK blobstor LVol as storage         |  Pri 2 /  Rel: (2025)*   |
+| 8    | GUI Mgmt Portal & Dashboard                   | Provision, manage, monitor Mayastor deployments with a RESTful GUI interface - @ parity with Shell & kubectl cmds | Pri 3 /  Rel: (Out-of-scope)    |
```
Comment (tiagolobocastro) [inline ROADMAP.md:40]: Should we remove the out of scopes?

File: ROADMAP.md [hunk 3]
```diff
@@ -11,44 +11,45 @@ OpenEBS follows a lean project management approach by splitting the development
...
+| 8.1  | GUI Mgmt Portal & Dashboard : On-Prem         | Mgmt portal & Dashboard deployed privately on-prem for air-gapped architectures                                     | Pri 3 /  Rel: (Out-of-scope)    |
+| 8.2  | GUI Mgmt Portal & Dashboard : In-Cloud SaaS   | Mgmt portal & Dashboard deployed as SaaS privately in-cloud for cloud enabled architectures                        | Pri 3 /  Rel: (Out-of-scope)    |
+| 8.3  | GUI Mgmt Portal & Dashboard : Global view     | Mgmt portal aggregated Global world view of all k8s clusters configured to contribute anonymized global stats     | Pri 3 /  Rel: (Out-of-scope)    |
+| 9    | Storage Encryption                            | Provision Encrypted data-at-rest volume via SPDK LVol layer - multi File system support (ext3, ext4, XFS, BRFS, SSDFS)  | Pri 1 /  Rel: (Q4 2024)    |
```
Comment (tiagolobocastro) [inline ROADMAP.md:44]:
```suggestion
| 9    | Storage Encryption                            | Provision Encrypted data-at-rest volume via SPDK LVol layer | Pri 1 /  Rel: (Q4 2024)    |
```

File: ROADMAP.md [hunk 4]
```diff
@@ -11,44 +11,45 @@ OpenEBS follows a lean project management approach by splitting the development
...
+| 13   | Native Object Store                           | An S3-compliant fast object store   integrated with SPDK LVstore/LVols Blobstor & HyperLocal-PV vols               | Pri 3 /  Rel: (Out-of-scope)    |
+| 14   | Replica topology                              | Replica distribution based on pool and node topologies                                                            | Pri 1 /  Rel: (Q3 2024)   |
+| 15   | Zoned-SSD support                             | Integrated Western Digital Team's Mayastor ZNS feature for very high performance vols                             | Pri 3 /  Rel: (Q4 2024)   |
+| 16   | NVMe-oF over RDMA support                     | Support for NVMe-oF over RDMA as transport for higher performance                                                | Pri 1 /  Rel: (Q4 2024)   |
+| 17   | Local PV CI and E2E hardening                 | Enhancing CI and E2E of Local PV engines with more tests                                                         | Pri 1 /  Rel: (Q4 2024)   |
```
Comment (tiagolobocastro) [inline ROADMAP.md:52]: We should add replicated hostpath as well?

**Review by tiagolobocastro — APPROVED** [review]

File: ROADMAP.md (hunk 1 above)
Comment (avishnu) [reply ROADMAP.md:23]: Ok, will add a status column.

File: ROADMAP.md (hunk 2 above)
Comment (avishnu) [reply ROADMAP.md:40]: We will remove them in the next update/iteration.

File: ROADMAP.md (hunk 4 above)
Comment (avishnu) [reply ROADMAP.md:52]: Ok

File: ROADMAP.md (hunk 1 above)
Comment (avishnu) [reply ROADMAP.md]: Added status to the last column itself, we need to re-structure the columns better in the future. Will take this up in a later PR.

File: ROADMAP.md (hunk 3 above)
Comment (avishnu) [reply ROADMAP.md]: Done

File: ROADMAP.md (hunk 4 above)
Comment (avishnu) [reply ROADMAP.md]: Added replicated hostpath and snapshot rebuild.

**Review by Abhinandan-Purkait — APPROVED** [review]

**Review by niladrih — APPROVED** [review]

**Pull request merged**